environment

```diff
@@ -226,6 +229,8 @@ _set AZUL_CART_API_IP_WHITELIST  # TODO: Remove when authentication is added
 
 _set azul_dss_query_prefix
 
+_set AZUL_FUSILLADE_HOSTNAME "auth.dev.data.humancellatlas.org"
```

Comment:
Other services endpoints are configured via a "endpoint" variable e.g., a base URL to which we append. This would allow for more flexible configuration.